component-samples/demo/aio/aio.env

```diff
@@ -4,7 +4,6 @@ aio_database_username=sa
 aio_database_password=
 aio_database_port=8081
 aio_database_driver=org.h2.Driver
-aio_database_allow_others=false
 aio_database_init_sql=./init.sql
 aio_database_new_device_sql=./new-device.sql
 owner_to0_rv_blob=http://localhost:8080?ipaddress=127.0.0.1
```

Comment:
It's better to pick the initialization value from .env than from a  .SQL file. We can remove the SQL command to update TO2_CONFIG table.